pkg/manifests/config.go

```diff
@@ -315,19 +325,18 @@ func (cfg *TelemeterClientConfig) IsEnabled() bool {
 	return true
 }
 
-func NewConfig(content io.Reader) (*Config, error) {
+func NewConfig(content io.Reader) (res *Config, err error) {
```

Comment:
I'm not a big fan of named return values as I find that they make the code less readable. Maybe this is only me but I don't feel either that this change is required here.